libhdf5/hdf5var.c

```diff
@@ -12,6 +12,20 @@
 #include <hdf5internal.h>
 #include <math.h> /* For pow() used below. */
 
+#undef REPORTCHUNKING
```

Comment:
How about using LOGGING instead of introducing a new preprocessor macro? This is what I do with debugging statements for dimensions, for example.

Reply:
Good idea